pkg/ddl/notifier/subscribe.go

```diff
@@ -91,18 +106,17 @@ type DDLNotifier struct {
 }
 
 // NewDDLNotifier initializes the global DDLNotifier. It should be called only
```

Comment:
I think we don't need to mention "called only once and before any RegisterHandler call.", because 
- it creates a new instance and call multiple times will create multiple instance.
- it's the receiver of RegisterHandler so it must be created first.

Reply:
Updated.